src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Entity/InstitutionConfigurationOptions.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddleware\ApiBundle\Configuration\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use JsonSerializable;
+use Surfnet\Stepup\Configuration\Value\Institution;
+use Surfnet\Stepup\Configuration\Value\ShowRaaContactInformationOption;
+use Surfnet\Stepup\Configuration\Value\UseRaLocationsOption;
+
+/**
+ * @ORM\Entity(
+ *      repositoryClass="Surfnet\StepupMiddleware\ApiBundle\Configuration\Repository\InstitutionConfigurationOptionsRepository"
+ * )
+ */
+final class InstitutionConfigurationOptions implements JsonSerializable
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="stepup_configuration_institution")
+     *
+     * @var Institution
+     */
+    public $institution;
+
+    /**
+     * @ORM\Column(type="boolean")
```

Comment:
Yes, I want these to be Doctrine types. I want it to return VOs or it's going to be too inconsistent for comfort.